client/src/views/files/FileContextMenu.vue

```diff
@@ -56,13 +56,12 @@
 
         <ion-item
           button
-          v-show="isDesktop()"
-          @click="onClick(FileAction.OpenInExplorer)"
+          @click="onClick(FileAction.Open)"
           class="group-item"
         >
           <ion-icon :icon="open" />
           <ion-label class="body">
-            {{ $t('FoldersPage.fileContextMenu.actionOpenInExplorer') }}
+            {{ $t('FoldersPage.fileContextMenu.actionOpen') }}
```

Comment:
`openInExplorer` sounds a bit weird, `open` is more generic and works for web too.